deploy/roles/jdbc/tasks/main.yml

```diff
@@ -1,4 +1,11 @@
 ---
+#Required for computer to make an ssl connection for postgres (since they sign their cert with letsencrypt)
+- name: Install cert utilities
+  yum: 
+    name: ca-certificates
+    state: latest
+  sudo: yes
```

Comment:
Thanks, I've updated all the `sudo`'s I've touched in ckan-installer to `become`'s but forgot to correct this one. Since `become_method` defaults to sudo, I've been leaving it off, but perhaps I should be explicitly adding it everywhere I use `become` just to be safe?